src/item.cpp

```diff
@@ -3198,21 +3202,6 @@ void item::calc_rot(const tripoint &location)
             time_duration spoil_variation = type->comestible->spoils * 0.2f;
             rot += factor * rng( -spoil_variation, spoil_variation );
```

Comment:
What am I missing here? Factor is a local variable, and you only update rot on the initial calc_rot when since is calendar::time_of_cataclysm. So where else is factor getting used and where is rot being updated?

Reply:
I'm afraid I don't understand the question fully.  calc_rot() gets gets called every turn under process_food.  The calculation gets updated every ten turns https://github.com/CleverRaven/Cataclysm-DDA/pull/25509/files/ab0ea357e668546c24018b182ced50074433543a#diff-f7ca61cf749eba039da961e8a7be1630R3177

I did notice that this `since` variable is no longer needed.

Reply:
Oh I think understand what you're saying now.  I did bork this.  Will fix.